CC-25662: Updated documents for Node.js v18 (#1857)
spryker/spryker-docs: docs/sdk/dev/development-tools/ts-linter.md

    link: docs/scos/dev/sdk/development-tools/tooling-config-file.html
---

{% info_block warningBox "No longer supported" %}

> Since `202304.0` release this tool is no longer supported and was replaced by `ESLint`. If you want to migrate on `ESLint`, see [Migration guide - Switch from TSLint to ESLint](/docs/scos/dev/front-end-development/{{site.version}}/migration-guide-switch-from-tslint-to-eslint.html).

Reviewer: ```suggestion
Since the `202304.0` release, this tool is no longer supported and has been replaced by `ESLint`. Follow the [migration guide](/docs/scos/dev/front-end-development/{{site.version}}/migration-guide-switch-from-tslint-to-eslint.html) to migrate to `ESLint`.
```

> To execute the TS Linter, do the following:

Reviewer: Please add a note at the beginning of the document that it has been migrated to ESLint since `202304.0`
